app/src/main/java/com/sun/cookbook/data/model/RecipeRecyclerView.kt

```diff
@@ -0,0 +1,18 @@
+package com.sun.cookbook.data.model
+
+data class RecipeRecyclerView(
```

Comment:
data class này có vẻ không thể hiện 1 recycler view 
nó thể hiện 1 đối tượng nào đó không liên quan tới recyclerview
```suggestion
data class Recipe(
```